EpsilonCCS/Inc/Data/BatteryData.h

```diff
@@ -23,7 +25,7 @@ enum BatteryPrechargeState
 
 struct BatteryData
```

Comment:
:+1: okay sounds good. I didn't realize that the structures matched the outgoing messages.